### PR TITLE
#540 Allow customizing assets or config files or both

### LINKED ISSUE #540: Use default assets/config files if these are not provided (Android/iOS)
When user provides only `src/android/AndroidManifest.xml` (without launch icons) or `src/ios/Default-info.plist` (without launcher assets), pick them from default assets in Substrate.

And also, the other way around, when user only provides icons, get the default manifest/plist from Substrate

### PR DESCRIPTION
**Pull request by jperedadnr**
This PR fixes #540, by allowing custom config files and/or custom assets

### CODE REVIEW

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5441026&bid=17842952)

File: src/main/java/com/gluonhq/substrate/Constants.java [hunk 1]
```diff
@@ -129,7 +129,9 @@
      * Supported files
      */
     public static final String PLIST_FILE = "Default-Info.plist";
+    public static final String ASSETS_FOLDER = "assets";
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/Constants.java:132]: prefix with ANDROID?

File: src/main/java/com/gluonhq/substrate/Constants.java [hunk 2]
```diff
@@ -129,7 +129,9 @@
...
     public static final String MANIFEST_FILE = "AndroidManifest.xml";
+    public static final String RES_FOLDER = "res";
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/Constants.java:134]: prefix?

File: src/main/java/com/gluonhq/substrate/Constants.java (hunk 1 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/Constants.java]: Ok, done

File: src/main/java/com/gluonhq/substrate/Constants.java (hunk 2 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/Constants.java]: Done

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5441026&bid=17842952)

**Pull request merged**